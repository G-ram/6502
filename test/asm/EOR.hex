a9 02 49 01 8d 00 01 
