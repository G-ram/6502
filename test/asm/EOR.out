A: 03
X: 00
Y: 00
P: 00010000
S: 00
PC: 607
100 : 03
101 : 00
